contracts/SwapFeeManager.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract SwapFeeManager is Ownable {
+    using SafeERC20 for IERC20;
+
+    address public immutable dexFeeWallet;
+    address public immutable burnFeeWallet;
+
+    event FeesSplit(uint256 dexFeeAmount, uint256 burnFeeAmount);
+
+    constructor(
+        address _dexFeeWallet,
+        address _burnFeeWallet
+    ) Ownable(_msgSender()) {
+        require(
+            _dexFeeWallet != address(0),
+            "dexFeeWallet must not be zero address"
+        );
+        require(
+            _burnFeeWallet != address(0),
+            "burnFeeWallet must not be zero address"
+        );
+
+        dexFeeWallet = _dexFeeWallet;
+        burnFeeWallet = _burnFeeWallet;
+    }
+
+    // Function to receive Ether
+    receive() external payable {}
+
+    /**
+     * @dev Splits the Ether balance in the contract into 75% for dexFeeWallet and 25% for burnFeeWallet.
+     */
+    function splitAndWithdraw() external onlyOwner {
+        uint256 totalBalance = address(this).balance;
+        require(totalBalance > 0, "No fees to split");
+
+        uint256 burnFeeAmount = (totalBalance * 25) / 100;
```

Comment:
can we set these percentages as const?

Reply:
yep we can, as constants are fixed at compile-time https://github.com/KomodoPlatform/etomic-swap/pull/8/commits/2af148a51ac89a127d8f77a95a62499c7c8775d0